src/imitation/algorithms/preference_comparisons.py

```diff
@@ -1163,7 +1172,7 @@ def __init__(
             else None
         )
 
-    def _make_data_loader(self, dataset: PreferenceDataset) -> data_th.DataLoader:
+    def _make_data_loader(self, dataset: data_th.Dataset) -> data_th.DataLoader:
```

Comment:
thanks, i remember this being an issue when splitting the preference dataset (you get a subset type) but did not want to change the type as I was not sure that was fine.